src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,28 @@
 package core.basesyntax;
 
+import core.basesyntax.machine.Machine;
+import core.basesyntax.models.Bulldozer;
+import core.basesyntax.models.Excavator;
+import core.basesyntax.models.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck1 = new Truck();
```

Comment:
It's bad practice to use numbers in variables names

Reply:
Thanks!) Done